dashboards/dashboard_2/README.md

```diff
@@ -9,34 +9,39 @@ https://github.com/user-attachments/assets/940ca4a7-65a4-40ea-a602-ea0f65e96746
 ## Features
 
 - Configure PV site parameters (latitude, longitude, capacity)
-- Select inverter type (No Inverter or Enphase)
+- Select inverter type (No Inverter, Enphase, Solis, GivEnergy, or Solarman)
 - Enphase API authentication flow (if applicable)
 - Run solar forecast
 - Visualize forecast results with interactive charts
 - Compare forecasts with and without recent PV data (if applicable)
 - Display raw forecast data and provide an option to download it as CSV
 
-## How to Run
+## Development Environment Setup
 
 1. Clone the repository and install all the dependencies in a virtual environment on a Linux System(or WSL):
    `pip install -e .` and `pip install -r requirements.txt`
 
-2. Set up environment variables (if applicable):
+2. Set up environment variables for your inverter (if applicable):
 
 - Create a `.env` file in your root directory
-- Add the following variables:
-  ```
-  ENPHASE_CLIENT_ID=your_client_id
-  ENPHASE_CLIENT_SECRET=your_client_secret
-  ENPHASE_API_KEY=your_api_key
-  ENPHASE_SYSTEM_ID=your_system_id
-  ```
+- Check out `.env.example` file and copy paste the contents of the file in the `.env` file that you just created
```

Comment:
could you add this file too?

Reply:
> could you add this file too?

The env example?

Reply:
> could you add this file too?

Didn't quite understand, could you please elaborate?

Reply:
`.env.example` ?

Reply:
> `.env.example` ?

That file's already there in the root directory of the project

Reply:
The readme says `Check out `.env.example` file and copy paste the contents of the file in the `.env` file that you just created`. How do i check out the .env.example file?

Reply:
> The readme says `Check out `.env.example`file and copy paste the contents of the file in the`.env` file that you just created`. How do i check out the .env.example file?

The env example file is in the Open-Source-Quartz-Solar-Forecast folder, it's already a part of the main branch of this repo, you can check it out there